src/main/xar-resources/data/docker/docker.xml

```diff
@@ -0,0 +1,96 @@
+        <orderedlist>
+          <listitem>
+            <para>
+              <code>release</code>: for the latest stable releases based on the master branch.</para>
+          </listitem>
+          <listitem>
+            <para>
+              <code>latest</code>: for last commit to the develop branch (within minutes of each commit).</para>
+          </listitem>
+        </orderedlist>
+      </sect2>
+    </sect1>
+
+    <sect1 xml:id="container-how-to">
+      <title>How to get started</title>
+      <para>First you need to download an images:</para>
+        <para><code>docker pull existdb/existdb:latest</code></para>
+      <para>Then you can start the container using that image:</para>
+        <para><code>docker run -it -d -p 8080:8080 -p 8443:8443 --name exist existdb/existdb:latest</code></para>
+    <sect2 xml:id="docker-what">
+        <title>What does this do?</title>
+        <para>You have just download and started eXist-db without launching an installer or having to provide java. More specifically:</para>
+        <itemizedlist>
+            <listitem>
+                <para><code>-it</code>: allocates a <literal>TTY</literal> and keeps <literal>STDIN</literal> open. This allows you to interact with the running Docker container via your console.
+                </para>
+            </listitem>
+            <listitem>
+                <para><code>-d</code>: detaches the container from the terminal that started it. So your container won't stop when you close the terminal.</para>
+            </listitem>
+            <listitem>
+                <para><code>-p</code>: maps the Containers internal and external port assignments (we recommend sticking with matching pairs). This allows you to connect to the eXist-db Web Server running in the Docker container.</para>
+            </listitem>
+            <listitem>
+                <para><code>--name</code>: lets you provide a name (instead of using a randomly generated one)</para>
+            </listitem>
+        </itemizedlist>
+        <para>The only required parts are <code>docker run existdb/existdb</code>.</para>
+        <para>For a full list of available options see the official <link xlink:href="https://docs.docker.com/engine/reference/commandline/run/">Docker documentation</link>.
+            You can now access your running instance by going go <literal>localhost:8080 </literal> inside your browser. To stop the container:</para>
+        <para><code>docker stop exist</code></para>
+    </sect2>
+    </sect1>
+    <sect1 xml:id="container-interact">
+        <title>Interacting with the running container</title>
+        <para>You can interact with a running container as if it were a regular Linux host. 
+            <important><para>GCR base images do not contain a shell this is by design. You can issue shell-like commands to the Java admin client, as we do throughout this readme, but you can't open the shell in interactive mode.</para></important> 
```

Comment:
I'd suggest either some punctuation (comma, new sentence) or getting rid of `this is` altogether.

* GCR base images do not contain a shell. This is by design.
* GCR base images do not contain a shell, this is by design.
* GCR base images do not contain a shell by design.